em/em.h

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
This filename should be updated once we come up with a better name for the sub-directory.

Also we this would apply to any residual-based type error estimation not just the electromagnetic ones.

Reply:
Name of the file has been changed to ree.h